backend/gateway/src/main/java/com/htec/gateway/controller/GatewayController.java

```diff
@@ -16,9 +19,9 @@ public GatewayController(GatewayService gatewayService) {
     }
 
     @RequestMapping(value = "/**")
-    public ResponseEntity<byte[]> handle(RequestEntity<byte[]> entity) {
+    public ResponseEntity<byte[]> handle(RequestEntity<byte[]> request, HttpServletRequest multipartRequest) throws IOException {
```

Comment:
change name to servletRequest, since that is going to be multipart only if file is uploaded